tests/Integration/ConnectionDbal2TestDouble.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Marein\LockDoctrineMigrationsBundle\Tests\Integration;
+
+use Doctrine\DBAL\Cache\QueryCacheProfile;
+use Doctrine\DBAL\Connection as DbalConnection;
```

Comment:
Alias not needed. Also see `ConnectionDbal3TestDouble` and `ConnectionDbal4TestDouble`.